buddyboss/buddyboss-platform
# #2548 - PHP8 compatibility issue fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2548 .

### How to test the changes in this Pull Request:

1. Install Buddyboss platform on a server where PHP 8 is installed
2. Do a network search
3. See it is working

### Proof Screenshots or Video

https://www.loom.com/share/3969ac2af8724d25a290791cd49024d3

### Other information:




### Changelog entry

> Network search PHP8 compatibility issue fixed.
